### DISCUSSION TITLE
Virus in Download

### DISCUSSION THREAD

**Discussion by Blitzboot** [QIDITECH/QIDI_Q1_Pro#102] [Q&A]
if i try to download the [Q1_Pro_V4.4.24.zip](https://github.com/QIDITECH/QIDI_Q1_Pro/releases/download/Q1_Pro_V4.4.24/Q1_Pro_V4.4.24.zip)
it shows me that there is a virus detected in the file..  

any one else had the issue?

Thanks

**Comment by QIDITECH** [comment]
We use frp for some data communication. Because it has the function of transmitting data, it is always identified as a virus by security software. But rest assured, he doesn't have any malicious features.
You can learn more by following the links below:
https://github.com/fatedier/frp/issues/2095
If you still feel that it is not secure, please do not update the firmware.
